service/src/main/java/uk/nhs/adaptors/gp2gp/mhs/MhsRequestBuilder.java

```diff
@@ -54,7 +55,7 @@
             .accept(APPLICATION_JSON)
             .header(ODS_CODE, fromOdsCode)
             .header(CONTENT_TYPE, APPLICATION_JSON_VALUE)
-            .header(INTERACTION_ID, MHS_OUTBOUND_INTERACTION_ID)
+            .header(INTERACTION_ID, MHS_OUTBOUND_EXTRACT_CORE_INTERACTION_ID)
```

Comment:
We'll need to provide a message id in the request headers since MHS won't tell us the message id it generates.

```suggestion
            .header(INTERACTION_ID, MHS_OUTBOUND_EXTRACT_CORE_INTERACTION_ID)
            .header(MESSAGE_ID, randomUuid)
```

Reply:
Nevermind, wrong location for this comment